packages/@react-spectrum/illustratedmessage/docs/IllustratedMessage.mdx

```diff
@@ -0,0 +1,97 @@
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Content} from '@react-spectrum/view';
+import {Heading} from '@react-spectrum/typography';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+```
+
+# IllustratedMessage
+
+<p>{docs.exports.IllustratedMessage.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['IllustratedMessage']} />
+
+## Example
+
+```tsx example
+<IllustratedMessage>
+  <svg width="150" height="103" viewBox="0 0 150 103" aria-hidden="true" role="presentation">
+    <path d="M133.7,8.5h-118c-1.9,0-3.5,1.6-3.5,3.5v27c0,0.8,0.7,1.5,1.5,1.5s1.5-0.7,1.5-1.5V23.5h119V92c0,0.3-0.2,0.5-0.5,0.5h-118c-0.3,0-0.5-0.2-0.5-0.5V69c0-0.8-0.7-1.5-1.5-1.5s-1.5,0.7-1.5,1.5v23c0,1.9,1.6,3.5,3.5,3.5h118c1.9,0,3.5-1.6,3.5-3.5V12C137.2,10.1,135.6,8.5,133.7,8.5z M15.2,21.5V12c0-0.3,0.2-0.5,0.5-0.5h118c0.3,0,0.5,0.2,0.5,0.5v9.5H15.2z M32.6,16.5c0,0.6-0.4,1-1,1h-10c-0.6,0-1-0.4-1-1s0.4-1,1-1h10C32.2,15.5,32.6,15.9,32.6,16.5z M13.6,56.1l-8.6,8.5C4.8,65,4.4,65.1,4,65.1c-0.4,0-0.8-0.1-1.1-0.4c-0.6-0.6-0.6-1.5,0-2.1l8.6-8.5l-8.6-8.5c-0.6-0.6-0.6-1.5,0-2.1c0.6-0.6,1.5-0.6,2.1,0l8.6,8.5l8.6-8.5c0.6-0.6,1.5-0.6,2.1,0c0.6,0.6,0.6,1.5,0,2.1L15.8,54l8.6,8.5c0.6,0.6,0.6,1.5,0,2.1c-0.3,0.3-0.7,0.4-1.1,0.4c-0.4,0-0.8-0.1-1.1-0.4L13.6,56.1z" />
+  </svg>
+  <Heading>No Results</Heading>
+  <Content>Try another search</Content>
+</IllustratedMessage>
+```
+
+## Content
+
+Each of these sections can be populated by providing the following components to your IllustratedMessage as children:
+`svg`, `Heading` (title), and `Content` (body).
+
+```tsx example
+function Example() {
+  let inlineStyle = {
+    fill: 'none',
+    'stroke-width': '3px'
+  };
+
+  return (
+    <IllustratedMessage>
+      <svg width="199" height="98" viewBox="0 0 199 97.7" aria-hidden="true" role="presentation" style={inlineStyle}>
+        <path d="M110.53,85.66,100.26,95.89a1.09,1.09,0,0,1-1.52,0L88.47,85.66" />
+        <line x1="99.5" y1="95.5" x2="99.5" y2="58.5" />
+        <path d="M105.5,73.5h19a2,2,0,0,0,2-2v-43" />
+        <path d="M126.5,22.5h-19a2,2,0,0,1-2-2V1.5h-31a2,2,0,0,0-2,2v68a2,2,0,0,0,2,2h19" />
+        <line x1="105.5" y1="1.5" x2="126.5" y2="22.5" />
+        <path d="M47.93,50.49a5,5,0,1,0-4.83-5A4.93,4.93,0,0,0,47.93,50.49Z" />
+        <path d="M36.6,65.93,42.05,60A2.06,2.06,0,0,1,45,60l12.68,13.2" />
+        <path d="M3.14,73.23,22.42,53.76a1.65,1.65,0,0,1,2.38,0l19.05,19.7" />
+        <path d="M139.5,36.5H196A1.49,1.49,0,0,1,197.5,38V72A1.49,1.49,0,0,1,196,73.5H141A1.49,1.49,0,0,1,139.5,72V32A1.49,1.49,0,0,1,141,30.5H154a2.43,2.43,0,0,1,1.67.66l6,5.66" />
+        <rect x="1.5" y="34.5" width="58" height="39" rx="2" ry="2" />
+      </svg>
+      <Heading>Drag and Drop Your File</Heading>
+      <Content>Select a File from your computer<br /> or Search Adobe Stock</Content>
```

Comment:
I removed their fake links from our example, it didn't seem like they added value.